### PR TITLE
Update print instructions with settings per part.

### PR DESCRIPTION
**Pull request by jonnolen**
making print settings match part names in source files and breaking them out per part.

### CODE REVIEW

**Review by darigovresearch — APPROVED** [review]
@jonnolen thanks for taking the time to make a pull request. Think separating it out like this makes it easier to understand.

Do any of the maintainers have any thoughts or require anything else before merging?

File: README.md
Comment (apopanda) [inline README.md:20]: Supports are recommended for the backing plate. We used to use two tiny plastic supports baked into the model, but later resigned to using full supports without those.

File: README.md
Comment (apopanda) [inline README.md:28]: The clip needs supports to print in the optimal orientation

**Comment by jenwilsonglia** [comment]
This pull request had inaccuracies. These have been fixed and merged with another pull request.

**Comment by jonnolen** [comment]
Stealing changes from people who have gone out of their way to improve something that is open source hides their contribution and is not good open source behavior.  

It took 7 months for anyone to look at these changes which were accurate when they were made. and then y'all took the changes and attributed them to another contributor.

**Comment by jonnolen** [comment]
There's no reason to go back and rewrite history for this. i just wanted to point out that it was a poor open source pattern of behavior, hopefully it doesn't happen in the future.

Like i said my changes were accurate when they were made 2022-07-15 and were accurate when they were approved in September of that year and then nothing happened with this PR until the underlying models were changed. it should have been merged back in 2022 and then the readme could have been updated with new settings when the models were changed.

**Comment by jenwilsonglia** [comment]
Ok. I actually went ahead and reverted the changes I made. I'm really sorry about that. I meant no disrespect, but I would like to honour your original pull request. Do you mind making the changes?

-Jen

March 14, 2023 10:34 AM, "Jon Nolen" ***@***.*** ***@***.******@***.***>)> wrote:
	There's no reason to go back and rewrite history for this. i just wanted to point out that it was a poor open source pattern of behavior, hopefully it doesn't happen in the future.

	Like i said my changes were accurate when they were made 2022-07-15 and were accurate when they were approved in September of that year and then nothing happened with this PR until the underlying models were changed. it should have been merged back in 2022 and then the readme could have been updated with new settings when the models were changed.

**Comment by jenwilsonglia** [comment]
Thanks Jon!

**Comment by jonnolen** [comment]
Thank you!

**Pull request merged**
